Content.Shared/Xenoarchaeology/Artifact/XAT/Components/XATDamageComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAT.Components;
+
+/// <summary>
+/// This is used for an artifact that is activated after a certain amount of damage is dealt.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, Access(typeof(XATDamageSystem))]
+public sealed partial class XATDamageComponent : Component
```

Comment:
Good example of why these need renaming. If you don't read "XAT" this is just a damage-tracking component, not something that triggers something.

XenoarchDamageTriggerComponent is much clearer.